### ISSUE TITLE
Update dependencies and publish latest pdfjs-dist v2.13.216

### ISSUE THREAD

**Issue by Lukenickerson** [bundled-es-modules/pdfjs-dist#24]
This repo is currently behind the latest version of pdfjs-dist by several minor versions: [v2.5.207](https://github.com/bundled-es-modules/pdfjs-dist/blob/master/package.json#L18) vs. [current v2.13.216](https://github.com/mozilla/pdfjs-dist/tags). We should update to the latest version of pdfjs-dist and other dependencies; hopefully in doing so, things are ironed out so that upgrades are just a matter of running `npm run build`.

I attempted to just update pdfjs-dist to the latest, but this gave an error during the build step due to optional chaining not supported in the older v1.x version of Rollup. I then updated rollup and all other dev depdencies (see [package.json](https://github.com/Lukenickerson/pdfjs-dist/blob/master/package.json)), but am getting this other error now:

```
./src/core/pdf.js → ./build/pdf.js...
[!] (plugin at position 5) SyntaxError: Unexpected token (1376:26) in pdfjs-dist\node_modules\pdfjs-dist\build\pdf.js
SyntaxError: Unexpected token (1376:26) in pdfjs-dist\node_modules\pdfjs-dist\build\pdf.js
    at Parser.pp$4.raise (pdfjs-dist\node_modules\acorn\dist\acorn.js:2757:13)
    at Parser.pp.unexpected (pdfjs-dist\node_modules\acorn\dist\acorn.js:647:8)
    at Parser.pp$3.parseExprAtom (pdfjs-dist\node_modules\acorn\dist\acorn.js:2196:10)
    at Parser.pp$3.parseExprSubscripts (pdfjs-dist\node_modules\acorn\dist\acorn.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (pdfjs-dist\node_modules\acorn\dist\acorn.js:2024:17)
    at Parser.pp$3.parseExprOps (pdfjs-dist\node_modules\acorn\dist\acorn.js:1966:19)
    at Parser.pp$3.parseMaybeConditional (pdfjs-dist\node_modules\acorn\dist\acorn.js:1949:19)
    at Parser.pp$3.parseMaybeAssign (pdfjs-dist\node_modules\acorn\dist\acorn.js:1925:19)
    at Parser.pp$3.parseMaybeConditional (pdfjs-dist\node_modules\acorn\dist\acorn.js:1954:28)
    at Parser.pp$3.parseMaybeAssign (pdfjs-dist\node_modules\acorn\dist\acorn.js:1925:19)
```

I suspect it's some incompatibility with a plugin, but I'm new to rollup, so don't know where to begin in troubleshooting this. The fork I'm working with is here: https://github.com/Lukenickerson/pdfjs-dist

**Comment by JBtje** [comment]
An update to the latest version of pdfjs-dist (v2.14.305 atm) would be nice, since the current version v2.5.207-rc1 is a little over 2 years old.
Furthermore, Laravel has switched to Vite by default, meaning that many project will be relying on this ESM build, since pdfjs-dist and Vite don't go together...

@robrez do you plan to update the package on short term? or accept PR?

**Comment by vlaraort** [comment]
+1 here

**Comment by robrez** [comment]
Okay, I've published a prerelease: [v2.16.15-alpha.1](https://www.npmjs.com/package/@bundled-es-modules/pdfjs-dist/v/2.16.105-alpha.1)

There were quite a few of rollup-related changes

- via https://github.com/bundled-es-modules/pdfjs-dist/pull/26

After that, bumping the pdfjs-dist version was pretty easy.. hopefully that will continue w/ subsequent upgrades needed

- via https://github.com/bundled-es-modules/pdfjs-dist/pull/27

Let me know how this pre-release works out for you, and barring any negative feedback I'll publish a normal tag.  

Closing this issue, please report any problems in a new issue
